pyproject.toml

```diff
@@ -1,58 +1,47 @@
-# Copyright (C) 2022 Obscurity Labs LLC. <admin@obscuritylabs.com> - All Rights Reserved
-# Unauthorized copying of this file, via any medium is strictly prohibited.
-# All rights reserved. No warranty, explicit or implicit, provided.
-# Proprietary and confidential.
-
 [tool.poetry]
-authors = ["Obscurity Labs"]
-description = "Project DarkMoon - Web API service"
 name = "darkmoon"
-version = "0.0.1"
+version = "0.0.2"
+authors = ["Obscurity Labs"]
+description = "Project DarkMoon - CLI tool"
+readme = "README.md"
 
 [tool.poetry.scripts]
 darkmoon = "darkmoon.main:cli_app"
 
 [tool.poetry.dependencies]
-MarkupSafe = "2.0.0"
-aenum = "^3.1.11"
-asyncio = "^3.4.3"
-fastapi = {version = "^0.95.0", extras = ["all"]}
-fastapi-mail = "^0.4.1"
-loguru = "^0.5.3"
-pydantic = {extras = ["dotenv"], version = "^1.8.2"}
-pydantic-jsonapi = "^0.11.0"
-pyjwkest = "^1.4.2"
 python = "^3.10"
-python-dotenv = "^0.15.0"
-python-slugify = "^4.0.1"
+fastapi = {version = "^0.95.0", extras = ["all"]}
 typer = {version = "^0.4.1", extras = ["all"]}
 types-requests = "^2.27.13"
+future = "^0.18.2"
+python-magic = "^0.4.27"
+requests = "^2.31.0"
 motor = "^3.0.0"
+httpx = "^0.24.1"
+uvicorn = "^0.22.0"
+pydantic = { extras = ["dotenv"], version = "^1.10.2" }
+pefile = "^2023.2.7"
 
-[tool.poetry.dev-dependencies]
-Faker = "^13.3.2"
-asgi-lifespan = "^1.0.1"
+[tool.poetry.group.dev.dependencies]
 black = "^22.6.0"
 commitizen = "^2.20.4"
 mkdocs = "^1.2.3"
 mkdocs-awesome-pages-plugin = "^2.7.0"
 mkdocs-material = "^8.2.5"
 mkdocs-minify-plugin = "^0.5.0"
 mkdocstrings = "^0.18.1"
-mypy = "^0.961"
-nest-asyncio = "^1.5.4"
+mypy = "^0.982"
 pre-commit = "^2.17.0"
-pydantic-to-typescript = "^1.0.6"
 pytest = "^7.1.2"
 pytest-cov = "^2.11.1"
+pytest-asyncio = "^0.21.0"
```

Comment:
Is this still needed when fastapi comes with built-in support for async pytest calls?